React/game/src/app/components/App/index.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+import Game from '../Screen/Game';
```

Comment:
We can use webpack's aliases for better import here! (Plus, they're used in our real projects so it's awesome to know about them) https://til.hashrocket.com/posts/38d2d28b6b-better-module-imports-with-webpack-aliases

Reply:
👍